element_lab/export/nwb.py

```diff
@@ -0,0 +1,84 @@
+from element_lab import lab
+
+
+def lab_to_nwb_dict(lab_key):
+    """
+    Generate a dictionary containing all relevant lab and institution info
+    :param lab_key: Key specifying one entry in element_lab.lab.Lab
+    :return: dictionary with NWB parameters
+    """
+    lab_info = (lab.Lab & lab_key).fetch1()
+    return dict(
+        institution=lab_info.get('institution'),
+        lab=lab_info.get('lab_name'),
+    )
+
+
+def project_to_nwb_dict(project_key):
+    """
+    Generate a dictionary object containing relevant project information
+        (e.g., experimental description, related publications, etc.).
+    :param project_key: Key specifying one entry in element_lab.lab.Project
+    :return: dictionary with NWB parameters
+    """
+    proj_info = (lab.Project & project_key).fetch1()
+    return dict(
+        experiment_description=proj_info.get('project_description'),
+        keywords=(lab.Project.Keywords() & project_key).fetch('keyword').tolist() or None,
+        related_publications=(lab.Project.Publication() & project_key).fetch('publication').tolist() or None
+    )
+
+
+def protocol_to_nwb_dict(protocol_key):
+    """
+    Generate a dictionary object containing all protocol title and notes.
+    :param protocol_key: Key specifying one entry in element_lab.lab.Protocol
+    :return: dictionary with NWB parameters
+    """
+    prot_info = (lab.Protocol & protocol_key).fetch1()
+    return dict(
+        protocol=prot_info.get('protocol'),
+        notes=prot_info.get('protocol_description')
+    )
+
+
+def elementlab_nwb_dict(lab_key=None, project_key=None, protocol_key=None):
```

Comment:
```suggestion
def element_lab_to_nwb_dict(lab_key=None, project_key=None, protocol_key=None):
```
For consistency, perhaps rename.